src/interoperability.md

```diff
@@ -155,11 +155,9 @@ serde = { version = "1.0", optional = true, features = ["derive"] }
 ```
 
 ```rust
-#[cfg(feature = "serde")]
-#[macro_use]
-extern crate serde;
+use serde::{Serialize, Deserialize};
 
-#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[derive(Serialize, Deserialize)]
```

Comment:
The removal of `cfg_attr` here misses the point of this example code, which is to show how to do a conditional (opt-in) serde impl.

I've put up #248 which removes the `extern crate` but keeps `cfg_attr`.